src/status_im/ui/screens/browser/views.cljs

```diff
@@ -95,6 +100,9 @@
          :local-storage-enabled                 true
          :start-in-loading-state                true
          :render-error                          web-view-error
+         :on-error                              #(when new?
+                                                   (re-frame/dispatch [:navigation-replace :open-dapp true])
```

Comment:
you need to move `open-new-url-error?` to `browser` in `app-db` in that case it will be reactive and you don't need `:navigation-replace :open-dapp`

Reply:
i think we don't need `new?` we should remove all errors, so if you open URL, then change it and got an error, and go back we should remove this too